src/components/pages/AdminHome/CourseDetails.js

```diff
@@ -16,6 +16,16 @@ const CourseDetails = props => {
             {course.start_date.split('-')[0]}/{course.start_date.split('-')[1]}{' '}
             @ {course.start_time} - {course.end_time}
           </p>
+          <p>
+            Syllabus:
+            {!course.syllabus_link ? (
+              <span className="black">{' currently unavailable'}</span>
```

Comment:
Another way to keep it clean, the beautiful ternary, great job very easy to understand what is going on